ff/serving/provider/offline.go

```diff
@@ -117,14 +117,17 @@ type OfflineStore interface {
 	RegisterPrimaryFromSourceTable(id ResourceID, sourceName string) (PrimaryTable, error)
 	CreateTransformation(config TransformationConfig) error
 	GetTransformationTable(id ResourceID) (TransformationTable, error)
+	UpdateTransformation(config TransformationConfig) error
 	CreatePrimaryTable(id ResourceID, schema TableSchema) (PrimaryTable, error)
 	GetPrimaryTable(id ResourceID) (PrimaryTable, error)
 	CreateResourceTable(id ResourceID, schema TableSchema) (OfflineTable, error)
 	GetResourceTable(id ResourceID) (OfflineTable, error)
 	CreateMaterialization(id ResourceID) (Materialization, error)
 	GetMaterialization(id MaterializationID) (Materialization, error)
+	UpdateMaterialization(id ResourceID) (Materialization, error)
```

Comment:
Why update materializations? Materializations are meant to be created ad-hoc and then deleted throughout the online store process.

Reply:
materializations change, no deletion now

Reply:
materializations change, no deletion now